Lib/test/libregrtest/save_env.py

```diff
@@ -208,32 +224,34 @@ def restore_threading__dangling(self, saved):
 
     # Same for Process objects
     def get_multiprocessing_process__dangling(self):
-        if not multiprocessing:
-            return None
+        multiprocessing_process = self.try_get_module('multiprocessing.process')
```

Comment:
Hm. The previous code checked the `multiprocessing.process` imported successful or not. Do we need keep it?

Reply:
The current code ignores ImportError. What do you mean?

```
try:
    import _multiprocessing, multiprocessing.process
except ImportError:
    multiprocessing = None
```

Reply:
Oh, sorry. Wrong comment in here. You raised the `SkipTestEnviroment` in here and catch it in `__enter__()`. So it is same as the old codes.